### ISSUE TITLE
Acebook issue

### ISSUE THREAD

**Issue by pawanrajcse** [vorpus/spadebook#1]
I am facing this problem with acebook
![acebook](https://user-images.githubusercontent.com/47554738/58234598-c83f4e80-7cf3-11e9-9117-d2d2d498725c.jpg)
, when i start the rails server, and visit localhost:3000 in browser, the page is rendered as blank, but there are no error issues. kindly help.

**Comment by albertngo1** [comment]
i'm getting this same issue @pawanrajcse

**Comment by pawanrajcse** [comment]
Yeah @albertngo1 , i tried by installing required gems such as (jquery, react, redox, etc..), nothing seems to work, There is some problem with respect to <% yield %> i think so..

But i feel this is a small silly error, if figured out, then the application will work fine.

**Comment by AmrMaghraby** [comment]
I faced the same problem and solved it after 4 hours of debugging.
First run
npm run heroku-postbuild
Then
rails s
